addons/product/views/product_views.xml

```diff
@@ -505,6 +505,18 @@
             </field>
         </record>
 
+        <record id="product_packaging_form_view2" model="ir.ui.view">
+            <field name="name">product.packaging.form.view2</field>
+            <field name="model">product.packaging</field>
+            <field name="mode">primary</field>
+            <field name="inherit_id" ref="product.product_packaging_form_view"/>
+            <field name="arch" type="xml">
+                <!-- Don't forward port in V >= saas-11.5 -->
```

Comment:
Not sure writing a comment here is the best way to avoid conflicts... BTW, does that mean that the other parts of the commit should still be forwardported?